services/web/pkg/theme/theme.go

```diff
@@ -65,14 +65,6 @@ var themeDefaults = KV{
 				"label":    "UnifiedRoleSecureView",
 				"iconName": "shield",
 			},
-			unifiedrole.UnifiedRoleFederatedViewerID: KV{
```

Comment:
I found no real usage of that.

Reply:
@JammingBen FYI

Reply:
Hmm are you sure? I think this is necessary for displaying the correct icons for the federated roles. Or am I missing something?

Reply:
Clarified with @fschade

Reply:
> Hmm are you sure? I think this is necessary for displaying the correct icons for the federated roles. Or am I missing something?

it uses the regular editor as example... the difference is the condition, the id is not used (and exposed)